src/hid/audio.h

```diff
@@ -23,15 +23,22 @@ class AudioHandle
         ERR,
     };
 
+    typedef const float* const* InputBuffer;
+    typedef float*              OutputBuffer;
```

Comment:
Why do you use a `float *` for both the interleaved and noninterleaved calbacks? The non-interleaved callback would need a `float **` or maybe even a `float * const *` (that is a const pointer to a non-const pointer if I'm not mistaken).

Reply:
This should be set up correctly... InputBuffer is for the Non-Interleaving, InterleavingInputBuffer is for the interleaving

Reply:
Ah I see that you chanegd it. Cool! Looking good!